### PR TITLE
added footer cta component

### PR DESCRIPTION
**Pull request by redevdesign**
Built a responsive call to action section just above the footer. It's a separate component that displays as part of the SiteFooter, so it renders on all pages,

### CODE REVIEW

File: components/blocks/FooterCallToAction.vue [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    >
+      <h2
+        class="text-white text-4xl leading-relaxed uppercase font-bold w-1/12 lg:w-1/2 pb-8 lg:p-0"
+      >
+        {{ block.text }}
+      </h2>
+
+      <div>
+        <BaseButton> {{ block.button[0].btnText }} </BaseButton>
+      </div>
```
Comment (drewclem) [inline components/blocks/FooterCallToAction.vue]: The base button needs an `:href` passed to it so it actually clicks through to the contact page

reference the button in the `HomepageHero`. should be a 1:1

Comment (drewclem) [reply components/blocks/FooterCallToAction.vue]: This button also needs to be `w-full` on mobile and `w-auto` from lg and up

**Review by drewclem — CHANGES_REQUESTED** [review]
Fantastic work on this!

Two _tiny_ changes that need to happen and then we can merge this bad boy in.

File: components/blocks/FooterCallToAction.vue (hunk 1 above)
Comment (redevdesign) [reply components/blocks/FooterCallToAction.vue]: :href is added.

Added responsive button width classes and added "block" class to BaseButton component.

Thanks for all of your help.

**Review by drewclem — APPROVED** [review]
Great work on this!! You're killing it.

There were some merge conflicts, courtesy of me doing unbranched work, so I resolved them just to get this out. Approved!!

merge this bad boy in

**Pull request merged**